src/Client.php

```diff
@@ -277,8 +277,8 @@ public function updateDocument($id, $data, $index, array $options = []): Respons
 
             $docOptions = $data->getOptions(
                 [
-                    'version',
-                    'version_type',
```

Comment:
Seems the right fix as this not documented anymore: https://www.elastic.co/guide/en/elasticsearch/reference/7.x/docs-update.html